app/javascript/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Greeting from './Greeting'
+import store from './redux/config/store'
+import { Provider } from 'react-redux'
+
+const App = () => {
+  return (
+      <Provider store={store}>
+          <Greeting />
+      </Provider>
+  )
+}
+
```

Comment:
[REQUIRED] You should have routes that are imported from react-router in app. And you should have a route. leading to Greeting component